specification/azureactivedirectory/resource-manager/Microsoft.Aadiam/preview/2020-07-01-preview/azureADMetrics.json

```diff
@@ -325,7 +325,12 @@
     "azureADMetricsConfig": {
       "description": "AzureADMetrics resource.",
       "type": "object",
-      "properties": {},
+      "properties": {
+        "provisioningState": {
+          "type": "string",
+          "description": "The provisioning state of the resource."
+        }
```

Comment:
this is where I have added the provisioningState property under object "properties", which is leading to a ModelValidationError.